src/mapping_navigation/scripts/TestGUI_helper.py

```diff
@@ -0,0 +1,31 @@
+import math
```

Comment:
What is this script used for?

Reply:
Was trying out the curve implementation based off my design in the report, when you run it plots the points which shows a curve. Currently just used to test out my theory and will also delete this script in my nextPR once I finish implementing the curve.